ansible/molecule/ec2_gpu/tests/test_default.py

```diff
@@ -8,29 +8,27 @@
 
 
 def test_nvidia_smi(host):
-    cmd = host.run("sudo nvidia-smi -L")
+    cmd = host.run("bash -c 'PATH=$PATH:/opt/bin sudo nvidia-smi -L'")
```

Comment:
clever :D

Reply:
@faiq indeed - I copied that from the default build tests;
```
def test_kubectl_avail(host):
    # the path is only set on interactive shell. SO lets append it here
    cmd = host.run("bash -c 'PATH=$PATH:/opt/bin type kubectl'")
    assert cmd.succeeded is True
    ```
 The original came from @fatz